test/integration/smoke/test_events_resource.py

```diff
@@ -161,21 +157,12 @@ def test_01_events_resource(self):
         virtual_machine.restore(self.apiclient)
         time.sleep(self.services["sleep"])
         virtual_machine.detach_volume(self.apiclient, volume)
-        volume.delete(self.apiclient)
-        self.cleanup.remove(volume)
         ts = str(time.time())
         virtual_machine.update(self.apiclient, displayname=ts)
-        virtual_machine.delete(self.apiclient)
-        self.cleanup.remove(virtual_machine)
         account_network.update(self.apiclient, name=account_network.name + ts)
-        account_network.delete(self.apiclient)
-        self.cleanup.remove(account_network)
+        virtual_machine.start(self.apiclient)
```

Comment:
@Pearl1594 I'm not completely aware of the new behaviour but idea of the test was to do a bunch of operations related to VM, network, volume etc and then check if the events for them have resourceid an resourcetype associated for them. Can we update the test in a way that similar check can be done? With the current change it is checking only for VM action event.

Reply:
I see your point and the point of such tests, but we are not checking for specific events anyway, just listing them. If the test is error prone, not specific and sensitive to environmental issues, I'd rather we simplify it and add more specific checks. Would you agree?

Reply:
@shwstppr I removed the delete resource operations because it did not matter in terms of the events being generated. As the test  doesn't check the type of event being generated for a resource, but rather just checking if events for a resource are generated. And the cleanup is happening via the teardown process.
Regarding the start operation, it was noticed that if a VM is restored - it loses its root disk, hence the test was failing on vmware during the deletion of the vm.

Reply:
@Pearl1594 Your change is correct in itself but I feel now we are verifying lesser set of actions. I would prefer if we can have only the change that fixes restore VM.
Like if earlier we were verifying 20 events, now we might only be doing 5 events.

Reply:
I've reverted the change, to keep only the relevant change to what this PR is attempting to fix. Thanks @shwstppr